docs/data/debugger.md

```diff
@@ -3,20 +3,36 @@ title: Debugging Analytics
 description: Troubleshoot and debug any data errors that may occur in the instrumentation process.
 ---
 
-Data validation is a critical step in the instrumentation process. Amplitude lets validate your event data via Amplitude's [User Lookup](#user-lookup) or using the [Instrumentation Explorer](#instrumentation-explorer) Chrome extension.
+Data validation is a critical step in the instrumentation process. Amplitude lets validate your event data by:
+
+- [Ingestion Debugger](#ingestion-debugger)
+- [User Lookup](#user-lookup)
+- [Instrumentation Explorer](#instrumentation-explorer) Chrome extension
 
 ### Before you begin: Instrument your events
 
-If you haven't instrumented any events, Amplitude's servers don't receive data, and no data is available in Amplitude itself. Amplitude strongly recommends that you create a test app to test and validate your first event data. See the [Quick Start Guide](https://help.amplitude.com/hc/sections/201146908) for more instructions and advice on getting started with Amplitude.
+**If you haven't instrumented any events, Amplitude's servers don't receive data, and no data is available in Amplitude itself**. Amplitude strongly recommends that you create a test app to test and validate your first event data. See [Plan your set up](../../analytics/plan-your-set-up/) for more instructions and advice on getting started with Amplitude.
```

Comment:
Amplitude's server won't receive any data

Reply:
It is not recommended to use future tense in dev docs. Future tense triggers Vale warnings and it was corrected by Casey from future tense to present tense in [this PR](https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/commit/263da76b971522da36b76e68cab08321344fcb90#diff-bb115d57637b32c42f3932a5a545a2eb2c0af922b0a1f19673b7023afa7d436e).